infrastructure/appinsights.tf

```diff
@@ -0,0 +1,10 @@
+resource "azurerm_application_insights" "appinsights" {
```

Comment:
how come this is here? its created by the webapp module unless you're using shared infra usually

Reply:
is needed by alert. and resource is not loaded until webapp module is resolved. `depends_on` did not work in module